README.md

```diff
@@ -1,23 +1,26 @@
 ![](https://github.com/asineth0/checkn1x/blob/master/icon_dark.png?raw=true)
 
-# checkn1x
+# checkn1x Reborn - bring back Checkn1x with latest Checkra1n version (0.12.3)
```

Comment:
You should probably remove this from the PR. It’s fine to have it on your own fork, but then create the PR from another branch at least.